Tools/Scripts/webkitpy/port/apple.py

```diff
@@ -113,6 +113,26 @@ def _generate_all_test_configurations(self):
                             configurations.append(TestConfiguration(version=version_name, architecture=architecture, build_type=build_type))
         return configurations
 
+    def all_baseline_search_paths(self, device_type=None):
+        paths = (
+            self.get_option("additional_platform_directory", [])
+            + self._compare_baseline()
+            + (
+                list(self._filesystem.glob(self._apple_baseline_path("*")))
```

Comment:
I would need to do some local spelunking to verify this concern, but our Apple specific baselines should be woven into our WebKit ones. So it should be something like:
```
apple-platform/ios-simulator
LayoutTests/platform/ios-simulator
apple-platform/ios
LayoutTests/platform/ios
```
instead of:
```
apple-platform/ios-simulator
apple-platform/ios
LayoutTests/platform/ios-simulator
LayoutTests/platform/ios
```
(which is what I'm afraid this change will do)

Reply:
You are, of course, technically correct. I guess given it's trying to provide all possible baseline directories it should really return a `set` rather than a `list`, which perhaps makes it clearer that there is no meaningful ordering here.

Reply:
Ah! That makes sense, I always expect our baseline search path to be ordered, returning a `set` would be fine, even just a comment that says "hey, order doesn't matter here, even though it usually does for this list"

Reply:
Right—it's not an unreasonable expectation, it's just we have no reasonable ordering for "does win7 or gtk come first?".